### PR TITLE
362 security insufficiently addressed in sdpi 10

### PR DESCRIPTION
**Pull request by ToddCooper**
Initial updates made to TF-1 and TF-3.  

Add text to TF-1A?

### CODE REVIEW

File: SDPi_Supplement/asciidoc/volume1/tf1-ch-10-sdpi-p.adoc [hunk 1]
```diff
@@ -786,8 +794,18 @@ No additional effectiveness requirements or considerations are identified for th
 
 // 10.5.4
 ==== Security Requirements & Considerations
-No additional security requirements and considerations are identified for this technical framework element beyond those provided by the  SDPi-P profile, and those specified in the _<<acronym_ses>> General Considerations_ section above.
+Security is foundational for all interactions between <<actor_somds_participant>> actors, with a clear distinction being made between information that may be exchanged outside of a secure connection (e.g., during network discovery transactions).
+Specific security technologies may vary based on the implementation technology being used, and will be detailed in the appropriate TF-2 technology specifications.
+All transactions indicate whether they require secure or unsecured connections.
+
+For the default <<acronym_sdpi_p>> connectivity technology, namely ISO/IEEE 11073 <<term_service_oriented_device_connectivity>>, TLS 1.2 support is required (See <<ref_ieee_11073_20701_2018>> and <<ref_ieee_11073_20702_2016>>).
```
Comment (d-gregorczyk) [inline SDPi_Supplement/asciidoc/volume1/tf1-ch-10-sdpi-p.adoc:801]:
```suggestion
For the default <<acronym_sdpi_p>> connectivity technology, namely ISO/IEEE 11073 <<term_service_oriented_device_connectivity>>, TLS 1.2 (or later versions) support is required (See <<ref_ieee_11073_20701_2018>> and <<ref_ieee_11073_20702_2016>>).
```

File: SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc [hunk 2]
```diff
@@ -147,7 +150,8 @@ NOTE: This section includes guidance and requirements that are focused on unique
...
 
 ==== Security Requirements & Considerations
-NOTE: This section includes guidance and requirements that are focused on unique *_Security_* requirements associated with associated technical framework element.  Note: in the context of risk management key properties, security is a state where information and systems are protected from unauthorized activities to a degree that the related risks to confidentiality, intgrity, and avaiability are maintained at an acceptable level throughout the lifecycle (see https://81001.org/concept/security[81001.org/security])
+NOTE: This section includes guidance and requirements that are focused on unique *_Security_* requirements associated with associated technical framework element.
+Note: in the context of risk management key properties, security is a state where information and systems are protected from unauthorized activities to a degree that the related risks to confidentiality, intgrity, and avaiability are maintained at an acceptable level throughout the lifecycle (see https://81001.org/concept/security[81001.org/security])
```
Comment (d-gregorczyk) [inline SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc:154]:
```suggestion
In the context of risk management key properties, security is a state where information and systems are protected from unauthorized activities to a degree that the related risks to confidentiality, intgrity, and avaiability are maintained at an acceptable level throughout the lifecycle (see https://81001.org/concept/security[81001.org/security])
```

File: SDPi_Supplement/asciidoc/volume2/dev-25/tf2-dev-25.adoc [hunk 3]
```diff
@@ -76,3 +76,25 @@ Model Metadata:: Expresses <<actor_somds_provider>> characteristics that are typ
...
+For additional guidance, see section <<vol1_clause_appendix_a_ses_considerations_and_requirements>>.
+
+==== Safety Requirements & Considerations
+No additional safety requirements or considerations are identified for this technical framework element beyond those specified in the _<<acronym_ses>> General Considerations_ section above.
+
+==== Effectiveness Requirements & Considerations
+No additional effectiveness requirements or considerations are identified for this technical framework element beyond those specified in the _<<acronym_ses>> General Considerations_ section above.
+
+==== Security Requirements & Considerations
+This transaction is intended to execute over *_SECURED_* connections.
```
Comment (d-gregorczyk) [inline SDPi_Supplement/asciidoc/volume2/dev-25/tf2-dev-25.adoc:95]:
```suggestion
This transaction is intended to execute over *_SECURED_* transmission channels.
```

File: SDPi_Supplement/asciidoc/volume2/dev-25/tf2-dev-25.adoc [hunk 4]
```diff
@@ -76,3 +76,25 @@ Model Metadata:: Expresses <<actor_somds_provider>> characteristics that are typ
...
+
+Protocol-specific security requirements for this transaction are detailed in the related TF-2 messaging technology appendix.
+For example, <<vol2_appendix_a_mdpws_messages>> for <<acronynm_mdpws>>-based communication.
+
+Appropriate security risk management shall be performed to ensure that no unacceptable harm results during *_SECURED_* exchange.
```
Comment (d-gregorczyk) [inline SDPi_Supplement/asciidoc/volume2/dev-25/tf2-dev-25.adoc:100]:
```suggestion
Appropriate security risk management shall be performed to ensure that no unacceptable harm results during *_SECURED_* message exchange.
```

File: SDPi_Supplement/asciidoc/volume2/dev-23/tf2-dev-23.adoc [hunk 5]
```diff
@@ -72,3 +72,28 @@ This message is sent
...
+==== Effectiveness Requirements & Considerations
+No additional effectiveness requirements or considerations are identified for this technical framework element beyond those specified in the _<<acronym_ses>> General Considerations_ section above.
+
+==== Security Requirements & Considerations
+This transaction is intended to execute over *_UNSECURED_* connections.
+
+Protocol-specific security requirements for this transaction are detailed in the related TF-2 messaging technology appendix.
+For example, <<vol2_appendix_a_mdpws_messages>> for <<acronynm_mdpws>>-based communication.
+
+Appropriate security risk management shall be performed to ensure that no unintended information is disclosed during *_UNSECURED_* exchange.
```
[the remark this answers is not in the document]
Comment (ToddCooper) [reply SDPi_Supplement/asciidoc/volume2/dev-23/tf2-dev-23.adoc]: Yes ... at least for now.  Eventually, we will also link in PKP ICS requirements to these SES clauses.  
Perhaps adding a statement to the Expected Actions or other existing clause would be OK too?  For now (pre-Public Comment) though, I just want to get SOMETHING like this in there.

**Review by d-gregorczyk — APPROVED** [review]

**Pull request merged**